src/domains/WinningLotto/index.js

```diff
@@ -0,0 +1,16 @@
+export class WinningLotto {
+  #lotto;
+  #bonusLottoNumber;
+
+  constructor(lotto, bonusLottoNumber) {
+    this.#lotto = lotto;
+    this.#bonusLottoNumber = bonusLottoNumber;
+  }
+
+  evaluateLotto(lotto) {
+    return {
+      matchingCount: this.#lotto.countMatchingLottoNumber(lotto),
+      isBonusNumberMatched: lotto.hasLottoNumber(this.#bonusLottoNumber),
+    };
+  }
```

Comment:
```suggestion
  evaluateLotto(lotto) {
    return {
      matchingCount: lotto.countMatchingLottoNumber(this.#lotto),
      isBonusNumberMatched: lotto.hasLottoNumber(this.#bonusLottoNumber),
    };
  }
```

이렇게 일관성있게 표현해주면 어떨까요!?